implementations/index.md

```diff
@@ -159,6 +162,11 @@ has a page describing how to emit conformant JSON.
 
 * [RestPack::Serializer provides examples](http://restpack-serializer-sample.herokuapp.com/) which demonstrate sample responses.
 * [Endpoints provides a fully working example API](http://github.com/endpoints/example/)
+* [http://hypermedia.cainosullivan.com](http://hypermedia.cainosullivan.com/) is a live JSON API service that displays a subset of the data from the 
+mythology.stackexchange.com site.
+  * [http://hypermedia.cainosullivan.com/v1/users](http://hypermedia.cainosullivan.com/v1/users?$prettify=yes)
+  * [http://hypermedia.cainosullivan.com/v1/posts](http://hypermedia.cainosullivan.com/v1/posts?$prettify=yes)
+  * [http://hypermedia.cainosullivan.com/v1/comments](http://hypermedia.cainosullivan.com/v1/comments?$prettify=yes)
```

Comment:
I'd rather not add this example unless it has some docs, description of available URLs, etc. We don't have room here to include a bunch of example URLs.